content/tutorial/tutorial.md

```diff
@@ -279,25 +279,25 @@ class Square extends React.Component {
 
   render() {
     return (
-      <button className="square" onClick={() => alert('click')}>
+      <button className="square" onClick={() => alert('klik')}>
         {this.props.value}
       </button>
     );
   }
 }
 ```
 
->Note
+>Uwaga
 >
->In [JavaScript classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), you need to always call `super` when defining the constructor of a subclass. All React component classes that have a `constructor` should start it with a `super(props)` call.
+>W [klasach javascriptowych](https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Classes) należy zawsze wywoływać specjalną metodę `super` w konstruktorze klasy potomnej. Dlatego wszystkie komponenty reactowe będące klasą, która mają zdefiniowany konstruktor powinny w nim na początku wywoływać `super(props)`.
 
-Now we'll change the Square's `render` method to display the current state's value when clicked:
+Teraz zmienimy kod w metodzie `render` komponentu `Square` tak, aby wyświetlał po kliknięciu wartość aktualnego stanu:
 
-* Replace `this.props.value` with `this.state.value` inside the `<button>` tag.
-* Replace the `() => alert()` event handler with `() => this.setState({value: 'X'})`.
-* Put the `className` and `onClick` props on separate lines for better readability.
+* Zamień `this.props.value` na `this.state.value` wewnątrz znacznika `<button>`.
```

Comment:
W *Lifting state up* używaliśmy przy `<input>` słowa element, więc może trzymajmy to spójnie?

```suggestion
* Zamień `this.props.value` na `this.state.value` wewnątrz elementu `<button>`.
```

Reply:
Element to coś, co trafia do drzewa DOM i jest na stronie. Znacznik to sposób zapisu (kod). Tutaj podmieniamy coś w kodzie, więc zostawiłbym "znacznika".